hail/python/hail/docs/functions/index.rst

```diff
@@ -89,6 +89,12 @@ These functions are exposed at the top level of the module, e.g. ``hl.case``.
 
     abs
     approx_equal
+    bit_and
+    bit_or
+    bit_xor
+    bit_lshift
+    bit_rshift
+    bit_flip
```

Comment:
Nice.  I think bit_not is more traditional.  I also agree we should have logical and arithmetic right shifts.